### PR TITLE
Add note about tx id, add examples for db config change, Closes #313, #314

### LINKED ISSUE #313: Hello World Tutorial documentation: index.js code could use an extra line
**Problem**
Received an 'could not connect to database' error after creating the index.js with the code from the tutorial copy pasted and starting the node.

**Solution**
Add the following line after `configDevnet.app.label = 'helloWorld-blockchain-app'; `:
`configDevnet.components.storage.password = '<password>'; // replace the <password> with the password for your lisk_dev database`
..or something similar.

This will avoid other people from stumbling upon the same error.

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #313, #314 

Additionally, this PR makes some other small improvements/fixes for the tutorials.

### CODE REVIEW

File: tutorials/cashback.md [hunk 1]
```diff
@@ -476,6 +478,8 @@ If the node accepted the transaction, it should respond with:
 
 To verify that the transaction was included in a block:
 
+> Use as `id` the id of your transaction object, that is posted to the node in the former step
```
Comment (4miners) [inline tutorials/cashback.md]: `previous step`

File: tutorials/cashback.md [hunk 2]
```diff
@@ -609,7 +612,7 @@ const CashbackTransaction = require('./cashback_transaction'); // require the ne
 
 configDevnet.app.label = 'cashback-blockchain-application';
 configDevnet.modules.http_api.access.public = true; // make the API accessible from everywhere
-//customConfig.modules.http_api.access.whitelist.push('1.2.3.4'); // example how to make the API accessible for specific IPs: add the host 1.2.3.4 to the whitelist of hosts
+//configDevnet.modules.http_api.access.whitelist.push('1.2.3.4'); // example how to make the API accessible for specific IPs: add the host 1.2.3.4 to the whitelist of hosts
```
Comment (4miners) [inline tutorials/cashback.md]: `specific IP addresses: add 1.2.3.4 IP address as whitelisted`

**Review by 4miners — CHANGES_REQUESTED** [review]

File: tutorials/hello-world.md [hunk 3]
```diff
@@ -74,6 +74,8 @@ Next, let's configure the application, to provide basic information about the ap
 const { Application, genesisBlockDevnet, configDevnet } = require('lisk-sdk'); // require application class, the default genesis block and the default config for the application
 
 configDevnet.app.label = 'helloWorld-blockchain-app'; // change the label of the app
+//configDevnet.components.storage.user = '<username>'; // In case you gave a different user than 'lisk' access to the database lisk_dev, you need to update the username in the config
```
Comment (RachBLondon) [inline tutorials/hello-world.md]:
```suggestion
//configDevnet.components.storage.user = '<username>'; // If you gave a different user than 'lisk' access to the database lisk_dev, you need to update the username in the config
```

Comment (Tschakki) [reply tutorials/hello-world.md]: Hm, that might be confusing. `lisk_dev` is the default database name, but the username and password refer to the database user who has access rights to the database. The default values are `lisk` for the database user and `password` for the password.

Comment (RachBLondon) [reply tutorials/hello-world.md]: Makes sense, I've adjusted just for a small amendment now.

**Review by RachBLondon — APPROVED** [review]

**Review by 4miners — APPROVED** [review]

**Pull request merged**